cloudflare/data_source_zone_dnssec.go

```diff
@@ -0,0 +1,98 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceCloudflareZoneDNSSEC() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceCloudflareZoneDNSSECRead,
+
+		Schema: map[string]*schema.Schema{
+			"zoneid": {
+				Type:             schema.TypeString,
+				Required:         true,
+				DiffSuppressFunc: zoneDiffFunc,
```

Comment:
What is the intention of using `zoneDiffFunc` here? The [function itself](https://github.com/cloudflare/terraform-provider-cloudflare/blob/b29fccfeba64c67757e00b4423490149f0700d92/cloudflare/resource_cloudflare_zone.go#L314-L325) is intended for performing unicode comparison of zone names.

Reply:
my mistake, I first started with the assumption that the zone id was the domain name. Forgot to clean up.